src/app/payments/helpers.ts

```diff
@@ -159,6 +161,37 @@ export const newCheckIntraledgerLimits = async ({
   })
 }
 
+export const newCheckTradeIntraAccountLimits = async ({
```

Comment:
is the `new` because it's replacing another method?

maybe a `v2` could be a better wording in that case. 

because new is very contextual and loose its context quickly

Reply:
We've started using the `new`/`New` prefix in a bunch of places as a placeholder while were transitioning things. This is meant to be temporary and `NewThing` will eventually revert back to just being `Thing`.